### ISSUE TITLE
Using createDrawerNavigation() causes Reanimated 2 Error

### ISSUE THREAD

**Issue by Stophface** [react-navigation/react-navigation#9451]
**Current Behavior**

I get this error

> Error: Reanimated 2 failed to create a worklet, maybe you forgot to add Reanimated's babel plugin?

> MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 change listeners added to [HasteMap]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)

> ERROR    Error: Requiring module "node_modules/@react-navigation/drawer/src/index.tsx", which threw an exception: Error: Reanimated 2 failed to create a worklet, maybe you forgot to add Reanimated's babel plugin?

With this code 

    import { createDrawerNavigator } from '@react-navigation/drawer';
    const Drawer = createDrawerNavigator();

**Expected Behavior**

Create a Drawer Navigation Object.

**How to reproduce**

    import { createDrawerNavigator } from '@react-navigation/drawer';
    const Drawer = createDrawerNavigator();

**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 | iOS 14.4
| @react-navigation/native       | 5.9.3
| @react-navigation/drawer       | 5.12.4
| react-native-reanimated        | 2.0.0
| react-native-gesture-handler   | 1.10.3
| react-native-safe-area-context | 3.2.0
| react-native-screens           | 2.18.1
| react-native                   | 0.63.4
| expo                           | -
| node                           | v15.8.0
| npm or yarn                    | 1.22.10

**Comment by raajnadar** [comment]
Use reanimated v1

**Comment by MnsrIl** [comment]
> Use reanimated v1

Unfortunately, it helps, but it's not the correct way to avoid this issue. 
Could any one suggest, how to fix this problem or not?

**Comment by raajnadar** [comment]
You need react navigation v6 to use the v2. At the time of my comment there was no v6 for react navigation.

**Comment by MnsrIl** [comment]
@raajnadar Oh, I got it. 
But, there is still a problem with react-native-reanimated 2, while you trying to use Drawer's Navigation along with Expo.
Also, I followed that setup, but unfortunately, it didn't help me with this error.
If you like, I can provide a repo with reproducing this bug. (It's happening, when you try to setup drawer navigation, using last Expo, React Native, react-navigation and react-native-reanimated packages.)
